docs/content/docs/dev/table/sql-gateway/overview.md

```diff
@@ -262,6 +262,18 @@ $ ./sql-gateway -Dkey=value
             <td>Integer</td>
             <td>The minimum number of worker threads for sql gateway service.</td>
         </tr>
+        <tr>
+            <td><h5>sql-gateway.security.mask-sensitive-options.enabled</h5></td>
+            <td style="word-wrap: break-word;">true</td>
+            <td>Boolean</td>
+            <td>Masks sensitive option values (e.g., passwords or tokens) in SHOW CREATE results to prevent accidental exposure.</td>
```

Comment:
I assume the default would be [hideSensitiveValues](https://github.com/apache/flink/blob/5753b098011f9b1c029364f263d7847c412e6024/flink-core/src/main/java/org/apache/flink/configuration/ConfigurationUtils.java#L181)

Would the names be in addition to the above or replace them